easy-ui-react/src/Calendar/Calendar.mdx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { ArgTypes, Canvas, Meta, Controls } from "@storybook/blocks";
+import { Calendar } from "./Calendar";
+import * as CalendarStories from "./Calendar.stories";
+
+<Meta of={CalendarStories} />
+
+# Calendar
+
+Calendar display a grid of days and allow users to select a single date.
+
+<Canvas of={CalendarStories.Default} />
+
+## Default Value
+
+A `<Calendar />` component allows users to set default selected date (uncontrolled).
+
+<Canvas of={CalendarStories.DefaultValue} />
+
+## Set minimum and maximun value
+
+Set minimum and maximum allowed date that a user may select. Dates outside the minimum and maximum value is disabled, and unreachable
+
+<Canvas of={CalendarStories.LimitAvailableDates} />
+
+## Show days outside of current month
+
+Display the days falling into the other months.
+
+<Canvas of={CalendarStories.ShowOutsideDays} />
+
+<Controls of={CalendarStories.ShowOutsideDays} include={["showOutsideDays"]} />
+
+## Set Date Availability
+
+Callback that is called for each date of the calendar. If it returns true, then the date is unavailable.
+
+<Canvas of={CalendarStories.DatesAvailability} />
+
+## Disabled Calendar
+
+Use `isDisabled` to disabled the calendar entirely.
+
+<Canvas of={CalendarStories.DisabledCalendar} />
+
+<Controls
+  of={CalendarStories.DisabledCalendar}
+  include={["isDisabled", "showOutsideDays"]}
+/>
+
+## Read-Only Calendar
+
+Use `isReadOnly` to make the calendar immutable.
+
+<Canvas of={CalendarStories.ReadonlyCalendar} />
+
+<Controls
+  of={CalendarStories.ReadonlyCalendar}
+  include={["isReadOnly", "showOutsideDays"]}
+/>
+
+## Controlled Selection
+
+For controlled selection, use `onChange` and `value`. Uncontrolled selection is possible with `defaultValue`. See associated code snippet.
+
+<Canvas of={CalendarStories.ControlledSelection} />
+
+## Invalid Selection
+
+An error message display when the selected value is invalid according to application logic.
```

Comment:
**note**: can we include text that lets the user know how to get an error message? i.e. "Select a weekend date to see an invalid selection."